docker-compose.yml

```diff
@@ -88,19 +105,26 @@ services:
       - .:/ProtectedPlanet
       - protectedplanet_node_modules:/ProtectedPlanet/node_modules
       - protectedplanet_bundler:/usr/local/bundle
-    network_mode: host
+    ports:
+      - "3035:3035"
```

Comment:
Please change to "3035" (without host port), so it won't conflict with host port. This is Jenkins test, no one will connect from the host.